colin-api/src/colin_api/models/business.py

```diff
@@ -81,7 +82,10 @@ class CorpStateTypes(Enum):
             TypeCodes.BC_COMP.value,
             TypeCodes.ULC_COMP.value,
             TypeCodes.CCC_COMP.value
-        ]
+        ],
+        LearBusinessTypes.EXTRA_PRO_A.value: [
+            TypeCodes.EXTRA_PRO_A.value
+        ],
```

Comment:
Yes, this is in line with https://github.com/bcgov/bcrs-shared-components/blob/0a0847e43d81566151e867d88c82f0408d2008cb/src/modules/corp-type-module/corp-type-module.ts#L92.